spinalcordtoolbox/scripts/sct_detect_pmj.py

```diff
@@ -256,6 +262,67 @@ def ifolder2tmp(self):
         self.curdir = os.getcwd()
         os.chdir(self.tmp_dir)  # go to tmp directory
 
+    def compute_cross_corr_3d(self, xrange=list(range(-10, 10)), xshift=10, yshift=10, zshift=10):  # Range of 20 is bit extreme?
```

Comment:
Our current PMJ test is performed on the PAM50 template:

https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/4f311bdd04c2ed76316cc5bd93ab9440b4dae598/testing/cli/test_cli_sct_detect_pmj.py#L17-L19

I assume this was done because it's the only image in `sct_testing_data` that actually contains the PMJ -- all other images are cropped to save time in processing, etc.

The problem is that the PAM50 template is already centered and perfectly symmetrical, so there is no RL shift needed! That makes it a little tricky to test this PR... we currently have no real-world images to test against.